src/new/connection.ts

```diff
@@ -60,6 +62,8 @@ class WalletConnection extends BaseConnection {
 
   streamWindowSize: number | undefined;
 
+  capabilities?: FullnodeCapability[];
```

Comment:
When the websocket connects the fullnode sends a capabilities message which contains some feature flags of the features enabled on the fullnode instance.

Currently only the `history-streaming` flag exists.